src/sync/sync-utils.ts

```diff
@@ -29,5 +31,16 @@ export async function findOrStartSync(
 	}
 
 	// Start sync
-	await sqsQueues.backfill.sendMessage({ installationId, jiraHost, startTime: fullSyncStartTime }, 0, logger);
+	await sqsQueues.backfill.sendMessage({ installationId, jiraHost, startTime: fullSyncStartTime, commitsFromDate }, 0, logger);
 }
+
+export const getCommitSinceDate = async (jiraHost: string, flagName: NumberFlags, commitFromDate?: string): Promise<string | undefined> => {
```

Comment:
/nit
`flagName` could be more strict typed as 
`SYNC_MAIN_COMMIT_TIME_LIMIT  | SYNC_BRANCH_COMMIT_TIME_LIMIT`

Reply:
yeah, agreed, it should just be the flags we do care about for this, which is the 2 in `NumberFlags` but you don't want your typings to include others if another one is added in the future.

Reply:
typed!